### DISCUSSION TITLE
Hyprland is the only thing that makes my computer easier to use with my ADHD

### DISCUSSION THREAD

**Discussion by jonzuk-real** [hyprwm/Hyprland#6311] [General]
I can get so much done because the way hyprland is set up just makes sense and its amazing I had a problem I think i fixed I was trying to use another DM for a bit and it was horrible i felt like i got punched in the face trying to figure out what to do compared to hyprland this might be the first open source project i donate to in the near future sorry if a post like this is unusual or not really welcome on github if it gets removed no hard feelings, im fairly new to github so i apologize again if these post are not normal on here but since there is discussion vs issues i figured i would share, I also would like to say on Nixos if you use flakes use the flake version of hyprland it works perfectly with no bugs 

Edit: I changed to Kernel 6.1 from 6.6 and it seems to have fixed the freezing problem i had (hopefully) I will keep trying to fix it because theres no other environment thats worth using hyprland is what computers should've been all along

**Comment by vaxerski** [comment]
yours truly :) enjoy
